media/server/main/include/MediaPipelineServerInternal.h

```diff
@@ -223,6 +223,11 @@ class MediaPipelineServerInternal : public IMediaPipelineServerInternal, public
      */
     bool m_wasAllSourcesAttachedCalled;
 
+    /**
+     * @brief Flag used to check if Eos has been set on this playback
+     */
+    bool m_isEos;
```

Comment:
Just a question - shouldn't we have m_isEos for each media type? What if we reach EOS for video, but we still need some data for audio?

Reply:
Yeah you are right, will change that.